src/main/kotlin/com/ajaxproject/warehouse/exception/MethodRateLimitExceededException.kt

```diff
@@ -0,0 +1,10 @@
+package com.ajaxproject.warehouse.exception
+
+import java.io.Serial
+
+class MethodRateLimitExceededException(message: String?) : RuntimeException(message) {
+    companion object {
+        @Serial
+        private const val serialVersionUID: Long = 106759828112291430L
+    }
```

Comment:
why? I don't think that we really need serialVersionUID in custom exception

Reply:
IDEA warned me to do this

Reply:
She is drunk!

Maybe, you should disable few inspections in IDEA. We should have serial version UID in serializable class, but not in all classes